media/starboard/starboard_renderer.cc

```diff
@@ -443,7 +432,7 @@ void StarboardRenderer::UpdateDecoderConfig(DemuxerStream* stream) {
 
     natural_size_ = decoder_config.natural_size();
 
-    player_bridge_->UpdateVideoConfig(decoder_config, "");
+    player_bridge_->UpdateVideoConfig(decoder_config, stream->mime_type());
```

Comment:
This is inside `#if 0`. Should we only ran this from this `#if 0` block, and leave the others related to `natural size` commented.